addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3513,6 +3513,17 @@ export class OdooEditor extends EventTarget {
                 node.before(fontNode);
                 node.replaceChildren(...fontNode.childNodes);
                 fontNode.appendChild(node);
+            } else if (node.nodeName === 'IMG' && node.getAttribute('aria-roledescription') === 'checkbox') {
+                const checklist = node.closest('ul');
+                const closestLi = node.closest('li');
```

Comment:
I had a look in the comparison view and it looks those lines were changed from `closestElement` to `\.closest`. Was there an issue when using `closestElement` there?

Reply:
The code is in `_cleanForPaste` and at this point node would not be in editor so we won't be able to use `closestElement` from 17.0 onwards.

Reply:
Right, now I think I remember we already talked about it, sorry!